flips/20220811-fungible-tokens-metadata.md

```diff
@@ -0,0 +1,210 @@
+# Fungible Tokens Metadata
+
+| Status        | Draft       |
+:-------------- |:---------------------------------------------------- |
+| **FLIP #1087**    | [1087](https://github.com/onflow/flow/pull/1087)|
+| **Author** | Álvaro Lillo Igualada (alvaro.lillo@dapperlabs.com) |
+| **Sponsor**   | Josh Hannan (joshua.hannan@dapperlabs.com)             |
+| **Updated**   | 2022-08-16                                           | 
+
+## Objective
+
+This proposal, in a similar fashion as the NFT Metadata, will make possible to create generic solutions that will interoperate through views rather than through hard-coded types. This will allow better interoperability between fungible tokens and external applications. 
```

Comment:
Probably include an embedded link to the NFT metadata flip and standard here

Reply:
make sure to address this

Reply:
A bazillion thanks for remembering this, there was a bunch of great suggestions made by @turbolent, including this point, that I thought I had merged but actually lost track of them at some commit.